content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -429,6 +429,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `CSIPersistentVolume`: Enable discovering and mounting volumes provisioned through a
   [CSI (Container Storage Interface)](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md)
   compatible volume plugin.
+- `CSIServiceAccountToken`: Enable CSI drivers to receive the pods' service account token that they mount volumes for. See design doc [Service Account Token for CSI Driver](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1855-csi-driver-service-account-token/README.md).
```

Comment:
Maybe we should point to csi docs instead?

Reply:
Good point. Linking to design docs should be treated as the last resort when we don't have docs for something important.

Reply:
We could merge “- `CSIServiceAccountToken`: Enable CSI drivers to receive the pods' service account token that they mount volumes for.” _for now_ and not close the related issue, pending the rest of the documentation (maybe a link to it?)

Reply:
once https://github.com/kubernetes-csi/docs/pull/383 merged, i can update the link.

Reply:
done